src/components/JumpToSection.vue

```diff
@@ -52,13 +52,13 @@ function toggleButton() {
   }
 }
 
-#main-content > div > div > div.header__secondary > nav .lux-button {
+#main-content div.header__secondary > nav .lux-button {
```

Comment:
Since I modified the structure in `App.vue`, these selectors needed to be updated